build.gradle

```diff
@@ -2,7 +2,7 @@ ext {
     coroutinesVersion = '1.5.2'
     wordPressUtilsVersion = '2.2.0'
     wordPressLoginVersion = '0.0.8'
-    gutenbergMobileVersion = 'v1.69.0-alpha2'
+    gutenbergMobileVersion = '4438-6ce40f20f16d2cb7b546cadd9e92b6f3c0cd8388'
```

Comment:
The changes here look good to me. Only question I have is if this will be changed next week when the `WordPress` app releases take place. Will they update to reflect the release tag?

Reply:
We update this ourselves after the gb-mobile repo branch is merged to trunk and we tag the release.

Reply:
Is this resolved now, with https://github.com/wordpress-mobile/WordPress-Android/pull/15805/commits/7ae04cf1e47e6f7d21c45c3d28a050811c0f5e6a?